pornography-hosts

```diff
@@ -15636,5 +15636,21 @@
 0.0.0.0 zzbabes.com
 0.0.0.0 zzpornpics.com
 0.0.0.0 zzztube.com
+
+0.0.0.0 ixiporn.com
```

Comment:
https://mypdns.org/my-privacy-dns/porn-records/-/issues/1437

Reply:
it's not the same ig

Reply:
`ig`? sorry but that I don't understand what means.

## Assumption
The following is based on the assumption that this list is "only" used by SB hosts.

You @defa02005 @Sinfonietta and everyone else be careful not to become dnmTX and StevenBlack sock puppets, just because they have serious personal issues. https://github.com/StevenBlack/hosts/issues/1773#issuecomment-940558163 and try to avoid giving the deserved credit https://github.com/StevenBlack/hosts/pull/1784 to me for my work over the past 2 years of maintaining that lists.

Appendix: Was looking for the issue where there dnmTX suddenly believed there was no point i blacklisting the actual content domain (cdn, thumbs, videos etc sub domains) unfortunate without luck.

## Question
IF = true, you did not copied all of these records (except `lustmaza.live`), how come you came around these records? As some of these are what I would consider very low quality sites.